wp-parsely.php

```diff
@@ -1105,7 +1105,7 @@ public function print_select_tag( $args ) {
 		if ( isset( $args['help_text'] ) ) {
 			echo '<div class="parsely-form-controls" data-has-help-text="true">';
 		}
-		if ( isset( $args['requires_recrawl'] ) ) {
+		if ( isset( $args['requires_recrawl'] ) && true == $args['requires_recrawl'] ) {
```

Comment:
Can each of the comparisons use strict types?

```suggestion
		if ( isset( $args['requires_recrawl'] ) && true === $args['requires_recrawl'] ) {
```